changelog.txt

```diff
@@ -1,7 +1,7 @@
 *** WooCommerce Shipping & Tax Changelog ***
 
 = 1.25.8 - 2021-xx-xx =
-* Tweak - Display better errors on checkout page when address fields are missing / invalid.
+* Tweak - Add support for new Jetpack 9.5 data connection.
```

Comment:
Let's include the 7.5 minimum version update in the changelog

Reply:
Good idea, done in 6e57114f